docs/02.API-REFERENCE.md

```diff
@@ -2322,6 +2334,79 @@ main (void)
 - [jerry_create_container](#jerry_create_container)
 - [jerry_container_type_t](#jerry_container_type_t)
 
+*New in version [[NEXT_RELEASE]]*.
+
+## jerry_container_operation
+
+**Summary**
+
+Perform container operation on the given operands (add, delete, set, etc.).
+
+*Note*:
+- Returned value must be freed with [jerry_release_value](#jerry_release_value) when it
+  is no longer needed.
+- This API function depends on a build option (`JERRY_BUILTIN_CONTAINER`) and can be checked
+  runtime with the `JERRY_FEATURE_MAP, JERRY_FEATURE_SET, JERRY_FEATURE_WEAKMAP, JERRY_FEATURE_WEAKSET`
+  feature enum values.
+  see: [jerry_is_feature_enabled](#jerry_is_feature_enabled).
+- The es.next profile enables this by default.
+
+**Prototype**
+
+```c
+jerry_value_t
+jerry_container_operation (jerry_container_operation_t operation,
+                           jerry_value_t container,
+                           jerry_value_t *arguments,
+                           uint32_t arguments_number)
+```
+
+ - 'operation' - container operation
+ - 'container' - this value
+ - 'arguments' - list of arguments
+ - 'arguments_number' - number of arguments
```

Comment:
AFAIK, we use backticks here (pleas check other API methods) instead of normal `'` characters.

Reply:
Yes you are right.